### PR TITLE
status for arch rebuild

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,13 +47,11 @@ <h4>Compiler Migration Deemed Sufficiently Complete!</h4>
                 <p>Ping "@conda-forge/core" as always, if you need help.
                 Thanks for your patience!</p>
                 <!-- Uncomment the following if you want the migration bar --->
-                <!---
```
Comment (scopatz) [inline index.html]: You also have to modify the template file

**Comment by CJ-Wright** [comment]
Not relevant for this PR but what is the best way to get more than one migration status thing on the page?

**Comment by scopatz** [comment]
It requires some modifications to the javascript to take an Array of URLs and then loop over those

**Comment by CJ-Wright** [comment]
Hmm ok, I was thinking of putting up on cf-graph which listed the active migrations so that the JS code could pull that rather than specifying it ourselves.

**Comment by scopatz** [comment]
Yeah, you could do that too, but this code would still need to loop over whatever structure you put on cf-graph.

**Comment by CJ-Wright** [comment]
Fair, this discussion is most likely better as an issue/independent PR

**Comment by scopatz** [comment]
Yeah, probably

**Pull request merged**